README.md

```diff
@@ -1,12 +1,33 @@
 # Northcoders News API
 
-Please create a .env.development file in the form of the env-example with your database name, please look at setup.sql for database names.
+This project hosts the backend for a news API. This includes files for database setup and seeding aswell as the server application. A hosted version of the project can be found [here](https://news-server-2i86.onrender.com/API). The linked API page contains a list of current endpoints and details on how they are used.
```

Comment:
it is somewhat redundant to call it the 'backend for a news API' since an API typically functions as the backend - I'd tweak this a bit to something like 'This project is the backend for a news service.'

I'd also not say that it hosts the backend, as that suggests that this is hosting the server, but we're using render to actually host it :)